src/controllers/expenses.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const expensesServices = require('../services/expenses.js');
+const userServices = require('../services/users');
+
+class ExpenseController {
+  getAll(req, res) {
+    const { userId, category, from, to } = req.query;
+
+    let vizibleExpenses = expensesServices.getAll();
+
+    if (userId) {
+      vizibleExpenses = vizibleExpenses
+        .filter(expense => expense.userId === +userId);
+    };
+
+    if (category) {
+      vizibleExpenses = vizibleExpenses
+        .filter(expense => expense.category === category);
+    }
+
+    if (from) {
+      vizibleExpenses = vizibleExpenses
+        .filter(expense => expense.spentAt >= from && expense.spentAt <= to);
+    }
+
+    res.statusCode = 200;
+    res.send(vizibleExpenses);
+  }
+
+  getOne(req, res) {
+    const id = req.params.expenseId;
```

Comment:
Still need to change(the same comment in the previous code review).
```suggestion
    const { expenseId } = req.params;
```